scripts/setup.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+. /etc/os-release
+
+# Install Software
+case "$ID_LIKE" in
+    rhel*)
+        # Install Ansible
+        type ansible >/dev/null 2>&1
+        if [ $? -ne 0 ] ; then
+            echo "Installing Ansible..."
+            sudo yum -y install epel-release >/dev/null
+            sudo yum -y install ansible >/dev/null
+        fi
+        ansible --version | head -1
+
+        # Install python-netaddr
+        python -c 'import netaddr' >/dev/null 2>&1
+        if [ $? -ne 0 ] ; then
+            echo "Installing Python dependencies..."
+            sudo yum -y install python36 python-netaddr >/dev/null
+	    sudo ln -s /usr/bin/python36 /usr/bin/python3
+        fi
+
+        # Install git
+        type git >/dev/null 2>&1
+        if [ $? -ne 0 ] ; then
+            echo "Installing git..."
+            sudo yum -y install git >/dev/null
+        fi
+        git --version
+        ;;
+    debian*)
+        # Update apt cache
+        echo "Updating apt cache..."
+        sudo apt-get update >/dev/null
+
+        # Install repo tool
+        type apt-add-repository >/dev/null 2>&1
+        if [ $? -ne 0 ] ; then
+            sudo apt-get -y install software-properties-common >/dev/null
+        fi
+
+        # Install Ansible
+        type ansible >/dev/null 2>&1
+        if [ $? -ne 0 ] ; then
+            echo "Installing Ansible..."
+            sudo apt-add-repository -y ppa:ansible/ansible >/dev/null
+            sudo apt-get update >/dev/null
+            sudo apt-get -y install ansible >/dev/null
+        fi
+        ansible --version | head -1
```

Comment:
Is there a way we can force an upgrade if ansible is not at the right version? Or better yet, freeze the version? It seems like lots of issues have arisen out of users having older versions and/or a specific bug in a particular version.

Reply:
I agree this is a problem. It's not clear how we should fix it, though.